solanum-tuberosums/bangazon_terminal_interface
# added a build_db method to create the db.sqlite3 file

## Associated Ticket
Set up the DB when your run main.py

## Description 
This adds a build_db() function to run with main.py if there isn't already a db.sqlite3 file. 
In methods.py I added the sql statements to build the db.
In main.py I added the logic using os.path that is now imported at the head of the file.

To test:
If you run main.py your db.sqlite3 file should not change.  
If you delete the db.sqlite3 file and run main.py again, it should create the file and load the application for you to use.

## All Tests Pass
